src/services/auth.service.js

```diff
@@ -0,0 +1,35 @@
+import nodemailer from 'nodemailer';
+
+const transporter = nodemailer.createTransport({
+  host: process.env.SMTP_HOST,
+  port: process.env.SMTP_PORT,
+  secure: true,
```

Comment:
Consider making the 'secure' option configurable based on the environment. For example, it might be set to 'false' for development environments where SSL/TLS is not used.